charts/coms/values.yaml

```diff
@@ -135,7 +132,7 @@ config:
     # DB_HOST: ~
     DB_PORT: "5432"
     # DB_POOL_MIN: "2"
-    # DB_POOL_MAX: "10"
+    DB_POOL_MAX: "5"
```

Comment:
opposite of what i was suggesting (LOL) but this actually make a lot of sense!
Hopefully Knex will queue the queries within this limit